bindings/gumjs/gumquickvalue.c

```diff
@@ -537,6 +538,42 @@ _gum_quick_args_parse (GumQuickArgs * self,
 
         break;
       }
+      case 'M':
+      {
+        GumMatchPattern * pattern;
+
+        if (JS_IsString (arg))
+        {
+          const char * str;
+
+          str = JS_ToCString (ctx, arg);
+          if (str == NULL)
+            goto propagate_exception;
+
+          pattern = gum_match_pattern_new_from_string (str);
+          JS_FreeCString (ctx, str);
```

Comment:
```suggestion

          JS_FreeCString (ctx, str);
```

To make it feel more "balanced" visually.